### DISCUSSION TITLE
One 'OSN' or many?

### DISCUSSION THREAD

**Discussion by philkomarny** [wgu-opensource/osmt#115] [Asking for Help] [answered]
Will the OSMT tool create/enable interoperation (open skill library) or just more skill silos? 

Empowering anyone/everyone to define a 'skill category' creates snowflakes where the intention was to instantiate a blizzard. If WGU and others want to really change the world, populate the OSN with some RSDs and let them be OPEN. 

For instance, if a POC is addressing the domain of Nursing, one of the more 'defined' domains, why wouldn't the OSN be pre-populated with RDS data so that pilots can run to make sense of this for business leaders? If the approach is to produce an open source tool that build proprietary outputs, I don't understand how this will work. 

Let's say we spend the time and craft the 100's of RSDs in the format you suggest in the CSV... how would all of that be merged if/when shared with the OSN?

Or maybe I'm missing something...

**Comment by drey-bigney** [comment, marked as answer]
Hi Phil,

These are great questions! While currently OSMT provides the ability to author, publish, and share RSDs and collections, it's just a small part of the emergent skills ecosystem.  We've got some enhancements in the works that will allow organizations to easily search and share RSDs and Collections beyond organizational boundaries.  Stay tuned!

**Comment by JohnKallies** [comment]
@philkomarny If you haven't seen this recent news, please take a look?
- https://www.wgu.edu/lp/skills-library.html#

**Reply by philkomarny** [reply]
I have! Thanks John.
